Maya/cicd/batch/buildMaya2Babylon.bat

```diff
@@ -0,0 +1,60 @@
+@echo off
+rem
+rem  buildBabylon2Maya.bat  -- Build and install dot net plugin
+rem 
+rem  usage:  buildBabylon2Maya
+rem  where:  install dll and nll assemblies to 2021 plug-ins directory:
+rem          located at: C:\Program Files\Autodesk\Maya(Version)\bin\plug-ins
+rem
+
+if "%1" == "" goto nomayaver
+
+set mayaversion=%1
+set mayalocation=%ProgramFiles%\Autodesk\Maya%mayaversion%
+
+set solutionfile=%~dp0..\..\Maya2Babylon.sln
+if not exist %solutionfile% goto nosolution
+
+set installdir=%mayalocation%\bin\plug-ins
+if not exist "%installdir%" goto noplugsdir
+
+set installeddll=%installdir%\Maya2babylon.dll
+if exist "%installeddll%" del /f "%installeddll%"
+set installedassembly=%installdir%\Maya2babylon.nll.dll
+if exist "%installedassembly%" del /f "%installedassembly%"
+
+set releasedir=%~dp0..\..\bin\Release\2020
+if exist "%releasedir%" del /f /q "%releasedir%"
+
+set assemblydir=%~dp0..\..\assemblies\2020
+if exist "%assemblydir%" del /f /q "%assemblydir%"
+
+msbuild %solutionfile% /v:quiet /t:Rebuild /p:Configuration=Release
```

Comment:
It might be good to check for errors after this. Usually goes something like this:
```suggestion
msbuild %solutionfile% /v:quiet /t:Rebuild /p:Configuration=Release
if errorlevel 1 goto :error
```

Same goes for the copying. Not sure if the errorlevel is right for msbuild though. Hopefully, it returns an error code when it fails.